client/src/commands.ts

```diff
@@ -42,14 +48,15 @@ export function cache(
 ): Callback {
   return (uris: DocumentUri[] = []) => {
     const activeEditor = vscode.window.activeTextEditor;
-    if (!activeEditor) {
+    const client = extensionContext.client;
+    if (!activeEditor || !client) {
```

Comment:
I have it returning silently when the language service doesn't exist here and in other places. Perhaps it should throw an error instead?